modules/numlock.nix

```
@@ -0,0 +1,19 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.xsession.numlock;
+in
+  {
+    options = {
+      xsession.numlock.enable = mkEnableOption "Numlock";
+    };
+
+    config = mkIf (cfg.enable) {
```

The parentheses are unnecessary
```suggestion
    config = mkIf cfg.enable {
```